moler/event.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, connection=None, till_occurs_times=-1, runner=None):
         self.till_occurs_times = till_occurs_times
         self._log_every_occurrence = True
         self.event_name = Event.observer_name
+        self._paused = False
```

Comment:
This should be threading Event not bool. Why? Since calling pause()/resume() is from test script thread (usually main thread) and data_received() is called by connection from connection thread.

Reply:
Discussed and agreed to keep Boolean.